### PR TITLE
Activating Open Collective

### PR DESCRIPTION
**Pull request by asood123**
This pull request adds backers and sponsors from your Open Collective https://opencollective.com/generator-jhipster ❤️
  
  It adds two badges at the top to show the latest number of backers and sponsors. It also adds placeholders so that the avatar/logo of new backers/sponsors can automatically be shown without having to update your README.md. [[more info](https://github.com/opencollective/opencollective/wiki/Github-banner)]. See how it looks on [this repo](https://github.com/apex/apex#backers).
We have also added a `postinstall` script to let people know after `npm|yarn install` that you are welcoming donations (optional). [[More info](https://github.com/OpenCollective/opencollective-cli)]
You can also add a "Donate" button to your website and automatically show your backers and sponsors there with our widgets. Have a look here: https://opencollective.com/widgets

  P.S: As with any pull request, feel free to comment or suggest changes. The only thing "required" are the placeholders on the README because we believe it's important to acknowledge the people in your community that are contributing (financially or with code!).

  Thank you for your great contribution to the open source community. You are awesome! 🙌
  And welcome to the open collective community! 😊

  Come chat with us in the #opensource channel on https://slack.opencollective.com - great place to ask questions and share best practices with other open source sustainers!

### CODE REVIEW

**Comment by sendilkumarn** [comment]
\o/ yay!!!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -302,3 +302,41 @@ Fix #1234
 ```
 
 [feature-template]: https://github.com/jhipster/generator-jhipster/issues/new?body=*%20**Overview%20of%20the%20request**%0A%0A%3C!--%20what%20is%20the%20query%20or%20request%20--%3E%0A%0A*%20**Motivation%20for%20or%20Use%20Case**%20%0A%0A%3C!--%20explain%20why%20this%20is%20a%20required%20for%20you%20--%3E%0A%0A%0A*%20**Browsers%20and%20Operating%20System**%20%0A%0A%3C!--%20is%20this%20a%20problem%20with%20all%20browsers%20or%20only%20IE8%3F%20--%3E%0A%0A%0A*%20**Related%20issues**%20%0A%0A%3C!--%20has%20a%20similar%20issue%20been%20reported%20before%3F%20--%3E%0A%0A*%20**Suggest%20a%20Fix**%20%0A%0A%3C!--%20if%20you%20can%27t%20fix%20this%20yourself%2C%20perhaps%20you%20can%20point%20to%20what%20might%20be%0A%20%20causing%20the%20problem%20(line%20of%20code%20or%20commit)%20--%3E
+
+
+## Financial contributions
```
Comment (MathieuAA) [inline CONTRIBUTING.md]: Maybe adding a link to this section in the summary is a good idea.

File: package.json [hunk 2]
```diff
@@ -59,6 +60,7 @@
     "lodash": "4.17.4",
     "meow": "3.7.0",
     "mkdirp": "0.5.1",
+    "opencollective": "^1.0.3",
```
Comment (MathieuAA) [inline package.json]: Can you change the version to a fixed one please? thanks

File: README.md [hunk 3]
```diff
@@ -58,3 +60,33 @@ Please read our [guidelines](/CONTRIBUTING.md#submitting-an-issue) before submit
...
+<a href="https://opencollective.com/generator-jhipster/sponsor/4/website" target="_blank"><img src="https://opencollective.com/generator-jhipster/sponsor/4/avatar.svg"></a>
+<a href="https://opencollective.com/generator-jhipster/sponsor/5/website" target="_blank"><img src="https://opencollective.com/generator-jhipster/sponsor/5/avatar.svg"></a>
+<a href="https://opencollective.com/generator-jhipster/sponsor/6/website" target="_blank"><img src="https://opencollective.com/generator-jhipster/sponsor/6/avatar.svg"></a>
+<a href="https://opencollective.com/generator-jhipster/sponsor/7/website" target="_blank"><img src="https://opencollective.com/generator-jhipster/sponsor/7/avatar.svg"></a>
+<a href="https://opencollective.com/generator-jhipster/sponsor/8/website" target="_blank"><img src="https://opencollective.com/generator-jhipster/sponsor/8/avatar.svg"></a>
+<a href="https://opencollective.com/generator-jhipster/sponsor/9/website" target="_blank"><img src="https://opencollective.com/generator-jhipster/sponsor/9/avatar.svg"></a>
+
+
```
Comment (MathieuAA) [inline README.md]: Can you remove the extra line here? thanks

File: package.json (hunk 2 above)
Comment (MathieuAA) [reply package.json]: Also: the `yarn.lock` needs an update.

**Comment by deepu105** [comment]
We also neef to update main website and advertise our enrollment into open collective

**Comment by jdubois** [comment]
Yes, let me just tell you about my current plan:

1. Do our next release :-)
2. Advertise for OpenCollective

Next week I will meet with Sean Larkinn in Paris, and with most of the JHipster French team, so it will be a good time to launch this.

**Comment by deepu105** [comment]
Nice, I will add a page and some updates on our main site soon, so that you
can merge it right before the announcement


Thanks & Regards,
Deepu

**Comment by deepu105** [comment]
@asood123 the open collective page is missing the JHipster logo, how can we add that?

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -53,3 +53,6 @@ If you are using JDL, please share that configuration as well.
 <!-- What OS are you on? is this a problem with all browsers or only IE8? -->
 
 - [ ] Checking this box is mandatory (this is just to show you read everything)
+
+<!-- Love generator-jhipster? Please consider supporting our collective:
```
Comment (deepu105) [inline .github/ISSUE_TEMPLATE.md]: change to "Love JHipster?"

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -302,3 +302,41 @@ Fix #1234
...
+We also welcome financial contributions in full transparency on our [open collective](https://opencollective.com/generator-jhipster).
+Anyone can file an expense. If the expense makes sense for the development of the community, it will be "merged" in the ledger of our open collective by the core contributors and the person who filed the expense will be reimbursed.
+
+
+## Credits
+
+
+### Contributors
+
+Thank you to all the people who have already contributed to generator-jhipster!
```
Comment (deepu105) [inline CONTRIBUTING.md]: change to "contributed to JHipster!"

**Comment by asood123** [comment]
@deepu105 An admin of the Collective can log in and change logo, background, etc. Currently, only @jdubois is listed as admin. He can invite other people to be admins.

**Comment by MathieuAA** [comment]
Thanks @deepu105 for making the last changes!! My internet connection was crappy at best last night, i couldn't finish.

**Comment by MathieuAA** [comment]
We shouldn't forget the `yarn.lock`/`package-lock.json` files

**Comment by pascalgrimaud** [comment]
@jdubois : did you forget this PR ?

**Comment by jdubois** [comment]
@pascalgrimaud oh yes, totally!! Let me have a look.
That's why our top priority at the moment should be to close tickets and PRs, we have far too many of those, and we get lost.

**Comment by jdubois** [comment]
This changes a lot of things, I need to have a look later today

**Comment by deepu105** [comment]
@jdubois if it helps I looked at this already and it looks all good

**Comment by jdubois** [comment]
I should merge it this morning - there are just a couple of things I need
to check

Le 1 déc. 2017 9:09 AM, "Deepu K Sasidharan" <notifications@github.com> a
écrit :

**Comment by jdubois** [comment]
In fact I need to copy some parts of this to http://www.jhipster.tech/sponsors/ so it's consistent, and also link to this page

**Comment by jdubois** [comment]
I'm going to reorder things here (to put sponsors higher), and also do links with http://www.jhipster.tech/sponsors/ )

**Pull request merged**